ehitchhiking-ui/src/app/shared/components/chat/chat.component.ts

```diff
@@ -0,0 +1,58 @@
+import {Component, OnInit} from '@angular/core';
+import {$} from 'protractor';
+
+interface ChatMessage {
```

Comment:
pls move to separate file in the shared/interfaces folder

Reply:
done